content/posts/2019-08-03---Going-from-ember-to-vue.md

```diff
@@ -146,14 +156,17 @@ I just wanted some examples of the major key pieces.
 First off, all of the markup is wrapped in a `template`.
 After that, you are writing normal HTML.
 You call other components just like `<AngleBracket />` components in Ember, but the differences (for now) is that you have to import them.
-This saves from potential name collision, as an npm package might have a component with the same name as one of yours.
-You could do `import SomeCustomComponent as Monkey from '@something/components';` and rename a component without any magic.
+This saves you from potential name collision, as an npm package might have a component with the same name yours.
+You could do `import SomeCustomComponent as Monkey from '@something/components';` and rename a component easily.
+Again, this is something in an RFC for Ember already but it cements a point I made in an earlier post that [the resolver is a bit too clever sometimes](/posts/a-few-thoughts-on-ember/#3-make-the-resolver-less-clever).
 
 ## Methods/Actions
 
-When calling a component, we have similar patterns as Ember.
-You need to append a `:` to properties you pass down to a component and `@` for events that map to functions (think `actions`).
+When using a component, we have similar patterns as Ember.
+You need to append a `:` to properties you pass down to a component.
+So, if my parent component has a property called `author` and I want to pass `author.name` down to a child component as `fullname`, it would look like this: `<AuthorInfo :fullname="author.name" />`.
```

Comment:
In Ember: 
```hbs
<AuthorInfo @fullname="author.name" />
```

Reply:
Personally I prefer Ember approach here (`{{}}` with help of helpers) instead of putting your JS inside a string which makes thing more difficult to debug.

Reply:
Updated to show an example of the two side-by-side